### DISCUSSION TITLE
Can't connect because the jitter keep be red

### DISCUSSION THREAD

**Discussion by Taglia74** [jamulussoftware/jamulus#3387] [Bug Investigation]
In the latest jamulus version I'm in trouble to connect with a lot of server. First of all I can't see the Any genere 1 page and I need to use Jamulus Explorer like work around, but worst is tha twhen I try to connect to some server I can't connect.....the strange things is that in connection time the jitter continuosly keep to be red and jamulus remain keep whaiting to connect. It happen only on some server (not other one).....the fact is that if one ip server address doesn't work it keep to not work also in different days. I have fiber connection with sky provider at 200Mb speed. May you help me about it?

**Comment by Taglia74** [comment]
Maybe is not why it happen but it seams that all the server that I can't connect are server with apple operation systema. I'm by windows pc.

**Comment by gilgongo** [comment]
What happens if you downgrade to [the previous version](https://github.com/jamulussoftware/jamulus/releases/tag/r3_10_0)? Does the trouble go away?

**Comment by ann0see** [comment]
Maybe this is related: https://github.com/orgs/jamulussoftware/discussions/3384

**Comment by ann0see** [comment]
Usually this is a network issue.
